site/includes/supported-by.hbs

```diff
@@ -1,4 +1,4 @@
-<section class="lnkbx">
+<section class="gc-contributors">
```

Comment:
Is the CSS "lnkbx" have style applied and used for other template?

Reply:
It is used in many <section> and <aside> tags